app/controllers/api/applications_controller.rb

```diff
@@ -1,74 +1,36 @@
 # frozen_string_literal: true
 
-class Api::ApplicationsController < Api::BaseController
```

Comment:
It looks like the only thing we had in `Api::BaseController` was the "bubbles" helper. I guess we don't need that for any of the different contexts for the Applications page, right?

cc/ @thomasmaas